examples/parteval/debug____.sf

```diff
@@ -0,0 +1,16 @@
+[type List[X] = Forall(n: Nat, Rec(n)(List => (Unit + (X, List))))]
```

Comment:
Is this file name on purpose?

Reply:
I think it wasn't supposed to be added, I'll either rename or remove it